### ISSUE TITLE
Vert.X 3.3.3 support?

### ISSUE THREAD

**Issue by cinterloper** [jtruelove/exovert#9]
Hello, will this be updated for vertx 3.3 series?

**Comment by jtruelove** [comment]
Yes, let me look at the changes required. What features are you currently using?

**Comment by cinterloper** [comment]
I am currently using the CRUD generator to create the basic implementation from loaded cassandra cql

I also use normal vertx codegen to generate a service-proxy implementation that consumes the CRUD methods exovert creates, although i write the interface for vertx-proxy-gen by hand

**Comment by jtruelove** [comment]
Cool always interesting to see how folks use things. Have you tried the
REST generation at all or does your proxy serve a different need?

**Comment by cinterloper** [comment]
I have not got around to trying that ... i have a bit of an internal debate as to weather i want to store binary objects in Cassandra or not

my application client mostly makes use of the event bus at the moment, although there is cause for uploading largeish files. 
if i decide to store those in Cassandra ill give the REST generator a go

**Comment by cinterloper** [comment]
thanks for this awesome project btw, its almost like magic!

**Comment by jtruelove** [comment]
I'm going to try and get to this, this weekend.
